src/super_gradients/training/transforms/depth_estimation/abstract_depth_estimation_transform.py

```diff
@@ -0,0 +1,22 @@
+import abc
+
+from super_gradients.training.samples import DepthEstimationSample
+
+
+class AbstractDepthEstimationTransform(abc.ABC):
+    """
+    Base class for all transforms for depth estimation sample augmentation.
+    """
+
+    @abc.abstractmethod
+    def apply_to_sample(self, sample: DepthEstimationSample) -> DepthEstimationSample:
+        """
+        Apply transformation to given depth estimation sample.
+        Important note - function call may return new object, may modify it in-place.
+        This is implementation dependent and if you need to keep original sample intact it
+        is recommended to make a copy of it BEFORE passing it to transform.
+
+        :param sample: Input sample to transform.
+        :return:       Modified sample (It can be the same instance as input or a new object).
+        """
+        raise NotImplementedError()
```

Comment:
don't we also want to have `__call__(self, sample: DepthEstimationSample) -> DepthEstimationSample:` ?

Reply:
Thre's actually no reason to keep apply_Sample for depth estimation (this is used in our other legact transforms for other tasks since we introduced their respective mixin, in order to handle the newly introduces sample classes).
Moved to __ call __ in dddf9d35e5954a6aa379d3ec3e3adc8fb7cf6f65